spec/spec_helper.rb

```diff
@@ -16,11 +16,12 @@
 
 Capybara.register_driver :apparition do |app|
   debug = !ENV['DEBUG'].nil?
+  headless = ENV['HEADLESS'].nil?
```

Comment:
if we're going to add this it should check for a specific value ('false') to disable headless rather than currently where doing something like `HEADLESS=true bundle exec rake` would actually disable headless